test-it/karma/utilities/TestHelpers.VireoRunner.js

```diff
@@ -3,10 +3,14 @@
     window.testHelpers = window.testHelpers || {};
 
     var rebootAndLoadVia = function (vireo, viaAbsolutePath) {
-        expect(viaAbsolutePath).toBeNonEmptyString();
+        // Jasmine Matchers library is not always ready in beforeAll so use jasmine core functions
+        expect(typeof viaAbsolutePath).toBe('string');
+        expect(viaAbsolutePath).not.toBe('');
```

Comment:
In our JS tests, we are using toBeNonEmptyString and toBeEmptyString and I thought that was the convention we wanted to follow !!??